test/AverageBalance.t.sol

```diff
@@ -22,7 +23,8 @@ contract AverageBalanceTest is AxiomTest {
 
         input = AxiomInput({ blockNumber: 4_205_938, _address: address(0x8018fe32FCFd3d166e8B4c4e37105318a84ba11d) });
 
-        querySchema = axiomVm.readCircuit("test/circuit/average.circuit.ts", "averageTest");
+        querySchema = axiomVm.readCircuit("test/circuit/average.circuit.ts", "aaaa");
+        console.logBytes32(querySchema);
```

Comment:
Should we add a test to `AxiomStd.t.sol` to check that the querySchema is actually what we're setting it to?